fxa-auth-db-server/test/backend/remote.js

```
@@ -616,11 +622,10 @@ module.exports = function(cfg, makeServer) {
 
             // Attempt to fetch a deleted session token
             return client.getThen('/sessionToken/' + user.sessionTokenId)
-          })
-          .then(function(r) {
-            assert(false, 'Fetching the non-existant sessionToken should have failed')
-          }, function(err) {
-            testNotFound(err)
+              .then(
+                () => assert(false, 'Fetching the non-existant sessionToken should have failed'),
+                () => assert('Fetching the non-existant sessionToken failed as expected')
+              )
```

Same as above re: catch-all handlers.